CoreGraphUtilities/src/au/gov/asd/tac/constellation/graph/utilities/hashmod/HashmodAction.java

```diff
@@ -202,15 +208,59 @@ private static void run(final GraphWriteMethods wg, final PluginInteraction inte
 
         if (createVertices) {
             numberSuccessful = 0;
+            float xVal;
+            float yVal;
+            float zVal;
+
+            // arrange new vertices
+            double widthGap = 2.0;
```

Comment:
looking at your algorithm, it seems are going for a grid like arrangement. If so, you can take advantage of the existing Grid arrangement at the end of the plugin.
e.g.
```
PluginExecutor.startWith(ArrangementPluginRegistry.GRID_COMPOSITE)
        .followedBy(InteractiveGraphPluginRegistry.RESET_VIEW)
        .executeNow(wg);
```

Reply:
Unfortunately, Core Arrangement Plugins is not currently a dependency and adding it would introduce cyclic dependency.  That's why it was added as inline code. Is there a way around the cyclic dependency?

Reply:
Looking at where Hashmod is, I'm actually starting to think it might be sitting in the wrong module. Given this is an import action, it (all the classes relating to Hashmod) might be better moved to Core Import Export Plugins. You would then be able to use the arrangement.
If you do that, if you're able to add an entry to `CHANGELOG.md` to note the class movement, that would be great

Reply:
Done